.github/workflows/iroha2-profiling-image.yml

```diff
@@ -11,10 +11,29 @@ env:
   IROHA2_RUSTFLAGS: -C force-frame-pointers=on
   IROHA2_FEATURES: profiling
   IROHA2_CARGOFLAGS: -Z build-std
+  DOCKER_COMPOSE_PATH: defaults
 
 jobs:
+  build_executor:
```

Comment:
why do you need executor to publish image?

Reply:
We need to keep executor in image and do not copy it from GitHub while deployment